documentation/src/main/wiki/GatewayMode.md

```diff
@@ -0,0 +1,41 @@
+# Gateway Mode
+
+The gateway mode provides an alternative way of processing requests, alternative to than presented in [[Knot Routing section|KnotRouting]].
+In this mode, Knots don't have to operate on Fragments. 
+
+## Example configuration
+
+Configuration of gateway mode takes place in the [[Server Configuration file|Server#How-to-configure]]. 
+
+```
+"customFlow": {
+  "routing": {
+    "GET": [
+      {
+        "path": "/customFlow/.*",
+        "address": "knotx.gateway.gatewayknot",
+        "onTransition": {
+          "next": {
+            "address": "knotx.gateway.requestprocessor"
+          }
+        }
+      }
+    ]
+  },
+  "responseProvider": {
+    "address": "knotx.gateway.responseprovider"
+  }
+}
+```
+
+Here, the gateway mode will work on paths beginning with `/customFlow`.
+First, a Knot, called a Gateway Knot, sets the Knot Context of the request. The rest of the routing configuration can be customized to your needs. 
+
+After the routing is over, the response is returned from a verticle called Response Provider. 
+
+//TODO an image of the routing simillar to this on Knot Routing page
```

Comment:
Unfortunately, I do not have access to the images in editable form. 
If you can provide it, I will create the needed image.